src/consts.ts

```diff
@@ -1,2 +1,4 @@
+import { createMuiTheme } from "@material-ui/core";
```

Comment:
It looks like this import is not necessary.

Reply:
OK